inst/app/app.R

```diff
@@ -0,0 +1,219 @@
+library(shiny)
+library(shinyWidgets)
+library(shinyjs)
+library(shinybusy)
+library(ggplot2)
+library(dplyr)
+library(rplanes)
+library(lubridate)
+
+# list module files and iterate sourcing them to use within the app.
+module_sources <- list.files(path = here::here("inst/app/modules"), full.names = TRUE)
```

Comment:
this is broken when you run `app_launcher()` ... im assuming you had it use `here::here()` for interactive development ... but make sure you fix. i had to manually change this to the line below to even get started on the review.